routes.js

```diff
@@ -18,28 +22,85 @@ router.post('/API/search', (req, res) => {
 
 
 
-import users from './loginfeature.js';// Here we import our read && csv function
-const users44 = [];
-users(users44);
 
-router.post("/login", (req, res) => {                                  // This is the post function, this will be activated when ever things that has been include in the form action in the htlm file.
-    // if then its activated this function will launch 
-    let usernametest = req.body.username;
-    let passwordtest = req.body.password;
-    console.log(usernametest)
-    console.log(passwordtest)
 
-    let user = users44.find(function (user) {
-        return user.username === usernametest && user.password === passwordtest;
+
+
+
+//New Page for forgot password This is the Current tasting page For Tokens login System. Dont touch it is hurting nobody.
+router.get("/forgot",verifyToken,(req,res)=>{
+
+    res.sendFile(path.resolve() + "/public/login/forgot.html");
+})
+router.post("/forgot",(req,res)=>{
+    console.log(req);
+    res.redirect("/");
+})
+
+
+
+//This is Carl && Shadi´s Dont touch
+router.get("/login/",(req,res)=>{                                   // this function defines what will be send when "/login" url is accesed  
```

Comment:
this is not supposed to be like this.
its supposed to be called index.html and then you dont need to make a route for it.